package.json

```diff
@@ -60,16 +60,16 @@
     "debug": "2.6.9",
     "dotenv": "10.0.0",
     "express": "4.19.2",
-    "express-graphql": "0.9.0",
+    "express-graphql": "0.12.0",
     "express-ipfilter": "0.3.1",
     "express-rate-limit": "3.2.0",
     "google-libphonenumber": "^3.2.25",
-    "graphql": "14.5.4",
+    "graphql": "15.8.0",
     "graphql-depth-limit": "1.1.0",
-    "graphql-middleware": "1.2.6",
+    "graphql-middleware": "4.0.3",
     "graphql-parse-resolve-info": "^4.12.3",
-    "graphql-relay": "0.5.4",
-    "graphql-tools": "4.0.5",
+    "graphql-relay": "0.7.0",
```

Comment:
I started initially by upgrading react-relay - that was a straightforward update because not much changed.
We had a patch for it that was not applicable anymore though. I replaced it with a new one and made sure to make the changes that in the new files. [See changes in patch 🔗 ](https://github.com/artsy/metaphysics/pull/5838/commits/ee44e5312d3d99fb06189d5266caff154d448c9c#r1689331684) 

▶️ **Next**